_grants/october-2014-uniform-guidance-implementation-a-series-of-dialogues.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
This file needs to be changed under policies_guidance folder. The grants, ERM, FDT, FMT etc. folders will ultimately be deleted as a part of refactoring.